starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -103,12 +104,18 @@
     'gettimeofday',
     'getifaddrs',
     'getaddrinfo',
+    'getpeername',
+    'getsockname',
+    'getsockopt',
     'gmtime_r',
     'inet_ntop',
     'listen',
     'lseek',
     'malloc',
     'posix_memalign',
+    'SbPosixSocketWaiterAdd',
+    'SbPosixSocketWaiterRemove',
+    'puts',
```

Comment:
Hm, this one seems a little suspect and unrelated to adding sockets. We generally don't have any other functions that interact with the console anywhere ( e.g. printf, putc etc ). What pulls this in ?